DevExtreme.AspNet.TagHelpers.Generator/PropTypeRegistry.cs

```diff
@@ -6,99 +6,79 @@
 namespace DevExtreme.AspNet.TagHelpers.Generator {
 
     static class PropTypeRegistry {
-        public const string CLR_TYPE_OVERRIDE_ATTR = "ClrTypeOverride";
-
-        public const string
-            CLR_BOOL = "bool",
-            CLR_DOUBLE = "double",
-            CLR_STRING = "string",
-            CLR_DATE = "DateTime",
-            CLR_OBJECT = "object",
-            CLR_ARRAY_STRING = "IEnumerable<string>",
-            CLR_ARRAY_OBJECT = "IEnumerable<object>",
-
-            SPECIAL_RAW_STRING = "sp_raw_string",
-            SPECIAL_DOM_TEMPLATE = "sp_dom_template";
 
         public static readonly IDictionary<string, string> OverrideTable = new Dictionary<string, string> {
```

Comment:
Hm, I suppose these registries are also be restructured, but you're right - each step should be completed, so will move this to the `PropertyTypeInfo`.